scripts/consensus.py

```diff
@@ -12,11 +16,19 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
     profiles : list
         list of paths pointing to normalized aggregated features
 
+    consensus_file_out : str
+        path where to write consensus signature file
+
     Returns
     -------
     pd.DataFrame
         concatenated normalized aggregated features
     """
+    # loading config file
+    aggregate_path_obj = Path(config)
```

Comment:
can you keep the nomenclature consistent?

i.e. `consensus_` instead of `aggregate_`. Using aggregate here is confusing